[56.971073151] 2021-11-14 14:10:28,049 - INFO - launcher: Starting sanity checks
[57.199001312] 2021-11-14 14:10:28,049 - INFO - launcher: Checking for Python 3.5+
[57.314157486] 2021-11-14 14:10:28,049 - INFO - launcher: Checking console encoding
[57.468891144] 2021-11-14 14:10:28,049 - INFO - launcher: Ensuring we're in the right environment
[482.824087143] 2021-11-14 14:10:28,474 - INFO - launcher: Required checks passed.
[483.067035675] 2021-11-14 14:10:28,475 - INFO - launcher: Optional checks passed.
[483.211040497] 2021-11-14 14:10:28,475 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[1.4699931144714355] 2021-11-14 14:10:29,462 - WARNING - musicbot.config | In config.py::MainThread(4659224064), line 121 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[1.4702138900756836] 2021-11-14 14:10:29,462 - INFO - musicbot.config | In config.py::MainThread(4659224064), line 132 in run_checks: Using i18n: config/i18n/en.json
[1.4722177982330322] 2021-11-14 14:10:29,464 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 225 in _setup_logging: Set logging level to INFO
[1.4741771221160889] 2021-11-14 14:10:29,466 - DEBUG - musicbot.json | In json.py::MainThread(4659224064), line 8 in __init__: Init JSON obj with config/i18n/en.json
[1.5148282051086426] 2021-11-14 14:10:29,506 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 90 in __init__: Starting MusicBot 897b1b8
[1.5151071548461914] 2021-11-14 14:10:29,507 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 96 in __init__: Loaded autoplaylist with 2527 entries
[4.4739680290222168] 2021-11-14 14:10:32,466 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 951 in on_ready: Connection established, ready to go.
[4.4741790294647217] 2021-11-14 14:10:32,466 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 342 in _cache_app_info: Caching app info
[4.6840231418609619] 2021-11-14 14:10:32,676 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 773 in _scheck_ensure_env: Ensuring data folders exist
[4.6876978874206543] 2021-11-14 14:10:32,679 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 789 in _scheck_server_permissions: Checking server permissions
[4.6879308223724365] 2021-11-14 14:10:32,679 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 793 in _scheck_autoplaylist: Auditing autoplaylist
[4.6881191730499268] 2021-11-14 14:10:32,680 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 797 in _scheck_configs: Validating config
[4.6882550716400146] 2021-11-14 14:10:32,680 - DEBUG - musicbot.config | In config.py::MainThread(4659224064), line 236 in async_validate: Validating options...
[4.6890420913696289] 2021-11-14 14:10:32,681 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 800 in _scheck_configs: Validating permissions config
[4.6894400119781494] 2021-11-14 14:10:32,681 - DEBUG - musicbot.permissions | In permissions.py::MainThread(4659224064), line 94 in async_validate: Validating permissions...
[4.6896789073944092] 2021-11-14 14:10:32,681 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 968 in on_ready: Connected: 909263827340779552/MusicBot_test#2817
[4.6904158592224121] 2021-11-14 14:10:32,682 - WARNING - musicbot.bot | In bot.py::MainThread(4659224064), line 996 in on_ready: Owner could not be found on any guild (id: 726637754771046400)

[4.6906812191009521] 2021-11-14 14:10:32,682 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 998 in on_ready: Guild List:
[4.6908550262451172] 2021-11-14 14:10:32,682 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 1001 in on_ready:  - 25時ディスコードで
[4.6910150051116943] 2021-11-14 14:10:32,683 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 1001 in on_ready:  - test server
[4.6911702156066895] 2021-11-14 14:10:32,683 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 1039 in on_ready: Not bound to any text channels
[4.6913402080535889] 2021-11-14 14:10:32,683 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 1065 in on_ready: Not autojoining any voice channels
[20.2624709606170654] 2021-11-14 14:10:48,254 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 2706 in on_message: 726637754771046400/Yoshie Suzuki#8679: !summon
[20.7024111747741699] 2021-11-14 14:10:48,694 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 733 in deserialize_queue: Deserializing queue for 856917545330933790
[20.8850760459899902] 2021-11-14 14:10:48,877 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 425 in get_player: Created player via deserialization for guild 856917545330933790 with 1 entries
[20.8852491378784180] 2021-11-14 14:10:48,877 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 536 in on_player_finished_playing: Running on_player_finished_playing
[20.8853340148925781] 2021-11-14 14:10:48,877 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 709 in serialize_queue: Serializing queue for 856917545330933790
[20.8858261108398438] 2021-11-14 14:10:48,877 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 1935 in cmd_summon: Joining 25時ディスコードで/作業場
[20.8869860172271729] 2021-11-14 14:10:48,879 - DEBUG - musicbot.entry | In entry.py::MainThread(4659224064), line 56 in get_ready_future: Created future for audio_cache/youtube-OlLEOBu3niI-UNDEAD_FORBIDDEN_RAIN_Music_MV.m4a
[20.8870820999145508] 2021-11-14 14:10:48,879 - FFMPEG - musicbot.player | In player.py::MainThread(4659224064), line 293 in _play: Creating player with options: -nostdin -vn audio_cache/youtube-OlLEOBu3niI-UNDEAD_FORBIDDEN_RAIN_Music_MV.m4a
[20.8924329280853271] 2021-11-14 14:10:48,884 - DEBUG - musicbot.player | In player.py::MainThread(4659224064), line 306 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x10b847790> using <discord.voice_client.VoiceClient object at 0x109707d90>
[20.8951151371002197] 2021-11-14 14:10:48,887 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 460 in on_player_play: Running on_player_play
[20.8966410160064697] 2021-11-14 14:10:48,888 - DEBUG - musicbot.bot | In bot.py::MainThread(4659224064), line 709 in serialize_queue: Serializing queue for 856917545330933790
[30.1349771022796631] 2021-11-14 14:10:58,127 - INFO - musicbot.bot | In bot.py::MainThread(4659224064), line 2706 in on_message: 726637754771046400/Yoshie Suzuki#8679: !disconnect
